backend/src/revisions/revisions.service.ts

```diff
@@ -219,4 +226,73 @@ export class RevisionsService {
       await this.revisionRepository.save(revision);
     }
   }
+
+  // Delete all old revisions everyday on 0:00 AM
+  @Cron('0 0 * * *')
+  async handleCron(): Promise<void> {
```

Comment:
I think we should rename these methods (`handleCron` and `handleTimeout`) to indicate that they're used in the context of revision cleanup. Maybe something like `handleRevisionCleanupTimeout` etc?